pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -86,6 +88,16 @@ func add(mgr manager.Manager, r *ReconcileOperConfig) error {
 		return err
 	}
 
+	// Likewise for the ConfigMap reconciler
```

Comment:
Rather than putting this here, please create an `Add()` function similar to `clusterconfig_controller.go` and add it to `add_networkconfig.go()`

Reply:
Changed.